beheer.md

```diff
@@ -185,65 +197,6 @@ Overdracht van de backlog en GitHub repository is een officieel moment en moet p
-9. Indien gewenst kan het aantal stacks on the fly gewijzigd worden in 'docker-compose.yml' en meteen geactiveerd worden door het commando in stap 8 weer opnieuw uit te voeren.
-
->_Ik vermoed dat stap 6 t/m 9 net door de beheerders wordt uitgevoerd maar alleen bij het runnen van de Docker containers. Indien dat het geval is dan kunnen deze stappen uit deze procedure worden verwijderd. Deze stappen moeten handmatig uitgevoerd kunnen worden maar zo mogelijk wordt dit geautomatiseerd._ 
-
-**Enkele belangrijke Docker commando's**
-
-Commando | Beschrijving 
---- | ---
-docker build -t [applicatie-naam] . | Compile een image van de applicatie in de folder.
-docker image ls | Lijst met alle lokale images.
-docker images | idem.
-docker images --all | Uitgebreide lijst met lokale images.
-docker login | Login op docker hub. Let op! Waar je in de browser omgeving als usernaam je emailadres gebruikt om in te loggen moet je hier echt je ID gebruiken.
-docker tag [image-naam] [username]/[repo-name]:[tag] | Koppelen van image aan te creeren docker hub repository.
-docker push [username]/[repo-name]:[tag] | Publiceren van image op docker hub. Nu kan de applicatie overal gedraaid worden waar docker geinstalleerd is. Als hij niet lokaal aanwezig is wordt deze op docker hub gezocht.
-docker run -p 4000:80 [image-naam] | start een lokale container running het image met de image-naam. Deze kan vervolgens gedraaid worden door http://localhost:4000. Je kunt uit de shell gaan d.m.v. CTRL+C.
-docker run -d -p 4000:80 [image-naam] | start een lokale container running het image met de image-naam in background mode. Deze kan vervolgens gedraaid worden in een browser door http://localhost:4000. De shell wordt in dit geval meteen verlaten.
-docker run -p 4000:80 [username]/[repository]:[tag] | Start een container vanaf de docker hub.
-docker container ls | Lijst met alle lokale containers.
-docker ps -a | idem.
-docker container stop [container-id] | Stop de container met id.
-docker rm [container-id] | Verwijder lokale container met id .
-docker rmi [image-id] | Verwijder lokale image met id.
-docker swarm init | Docker o.a. configureren voor het gebruik van een docker-compose file.
-docker stack deploy -c docker-compose.yml getstartedlab | deploy de docer-compose.yml.
-docker service ls | Haal de gegevens van de service.
-docker service ps getstartedlab_web | Haal de gegevens op van alle tasks die de service draaien.
-docker stack rm getstartedlab | Afsluiten van de applicatie.
-docker swarm leave --force | Afsluiten van de swarm.
+## Klankbordgroepen
+Voor de afstemming met stakeholders is het noodzakelijk klankbordgroepen te vormen. Zowel gemeenten/klanten als leveranciers moeten hier zitting in hebben. Om te voorkomen dat net als met de huidige werkgroepen voornamelijk leveranciers deelnemen moet hier speciaal aandacht voor zijn. Een mogelijkheid kan zijn aparte klankbordgroepen voor zowel gemeenten als leveranciers.
```

Comment:
Schets ook de verantwoordelijkheden van de leden van de klankbordgroepen, zoals:
* Het monitoren van de ingebrachte issues en PR's;
* Het zo mogelijk beantwoorden van vragen;
* Het mede bepalen van de prioriteiten van de ingebrachte issues en dientengevolge ook de roadmap;
* ....

Wellicht moet hier ook aangegeven hoe vaak per jaar we de klankbordgroepen fysiek bij elkaar willen roepen.

Reply:
Taken en verantwoordelijkheden van de (leven van de) klankbordgroepen ook opgenomen.